url.bs

```diff
@@ -177,6 +177,9 @@ U+003E (>), U+003F (?), U+0060 (`), U+007B ({), and U+007D (}).
 <a>path percent-encode set</a> and U+002F (/), U+003A (:), U+003B (;), U+003D (=), U+0040 (@),
 U+005B ([), U+005C (\), U+005D (]), U+005E (^), and U+007C (|).
 
+<p>The <dfn>fragment percent-encode set</dfn> is the <a>C0 control percent-encode set</a> and
```

Comment:
How about we try to define path percent-encode set as a superset of fragment percent-encode set, for parity with userinfo->path.

Reply:
Makes sense to me. Poked at it in https://github.com/whatwg/url/pull/347/commits/f258f4dd4dfba66c7eed17534752065eed8a748a. WDYT?